client/signup/steps/site-topic/index.jsx

```diff
@@ -60,6 +60,9 @@ class SiteTopicStep extends Component {
 
 	onSiteTopicChange = value => {
 		this.setState( { siteTopicValue: value } );
+		if ( this.props.flowName === 'onboarding-dev' ) {
```

Comment:
I assume these changes are unrelated?

Reply:
I think he's doing this to test updating the topic (and the preview) onChange, not just onSubmit.